src/upnp_xml.cc

```diff
@@ -50,7 +50,7 @@ UpnpXMLBuilder::UpnpXMLBuilder(const std::shared_ptr<Context>& context,
     for (auto&& entry : this->config->getArrayOption(CFG_IMPORT_RESOURCES_ORDER)) {
         auto ch = MetadataHandler::remapContentHandler(entry);
         if (ch > -1) {
-            orderedHandler.push_back(ch);
+            orderedHandler.push_back(std::move(ch));
```

Comment:
That's an integer. Is move really changing anything?

Reply:
Nope. Will fix.

Reply:
Fixed.